style.css

```diff
@@ -7,3 +7,125 @@
 html {
   font-size: 62.5%;
 }
+
+body {
+  font-family: "Inter", sans-serif;
+  width: 37.5rem;
+  margin: 0 auto;
+  line-height: 1;
+  background-color: #e5e5e5;
+}
+
+.bg-color {
+  background-color: #fff;
+}
+
+/* HEADER */
+
+.main-header {
+  text-align: center;
+}
+
+.notification-bar {
+  height: 4.4rem;
+  width: 100%;
+}
+
+.notification-img {
+  width: 100%;
+}
+
+.main-nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 4.8rem;
+}
+
+.menu {
+  font-size: 2.4rem;
+}
+
+/* HERO SECTION */
+
+.section-hero {
+  background-image: url(header-illsutration-mobile@2x.png);
+  background-size: cover;
+  padding: 9.6rem 0;
+  position: relative;
+  text-align: center;
+}
```

Comment:
- [ ] Please use `flexbox` instead of `position` because this is a requirement so kindly use `flexbox` instead of `position` :+1: